### PR TITLE
Support fcitx5 dbus interface

### PR DESCRIPTION
**Pull request by blahgeek**

### CODE REVIEW

**Comment by yangdaweihit** [comment]
这是来自QQ邮箱的假期自动回复邮件。您好，我已收到您的邮件。

**Comment by blahgeek** [comment]
question: is there anyway to generate readme.org from fcitx.el docstring? or should I do it manually?

File: fcitx.el [hunk 1]
```diff
@@ -577,6 +583,28 @@ Re-run the setup function after `fcitx' is started.")))
                        "GetCurrentState")
      2))
 
+(defun fcitx--activate-dbusv5 ()
```
Comment (cute-jumper) [inline fcitx.el:586]: I'll suggest adding a separator to make the name easier to read:
```suggestion
(defun fcitx--activate-dbus-v5 ()
```

Please also update the others accordingly

**Comment by cute-jumper** [comment]
> question: is there anyway to generate readme.org from fcitx.el docstring? or should I do it manually?

Just personal preference: I prefer editing org files directly so what I would do is the opposite: edit the README.org and then convert that into the comments (which is why [org2elcomment](https://github.com/cute-jumper/org2elcomment) exists).

File: fcitx.el (hunk 1 above)
Comment (blahgeek) [reply fcitx.el]: done

**Comment by cute-jumper** [comment]
Thanks!

**Pull request merged**